util/versionutil/version.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
 
-	"github.com/pingcap/tidb-dashboard/pkg/utils/distro"
+	"github.com/pingcap/tidb-dashboard/util/distro"
 )
 
 type Info struct {
```

Comment:
I suspect this util can only be used in TiDB Dashboard instead of TiDB Dashboard variants, as not all variants needs to specify a PD version. Is it possible to make it more general?